src/sentry/integrations/vercel/integration.py

```diff
@@ -131,6 +137,20 @@ def get_pipeline_views(self):
 
         return [identity_pipeline_view]
 
+    def get_configuration_metadata(self, external_id):
+        # If a vercel team or user was already installed on another sentry org
+        # we want to make sure we don't overwrite the existing configurations. We
+        # keep all the configurations so that if one of them is deleted from vercel's
+        # side, the other sentry org will still have a working vercel integration.
+        try:
+            integration = Integration.objects.get(external_id=external_id, provider=self.key)
+        except Integration.DoesNotExist:
+            # not sure if we care to log this, maybe we should just log if
```

Comment:
@MeredithAnya This just means the installation isn't installed, yea? Don't think we need to log this.

Reply:
do we want to log when someone is setting up a second install for the same vercel team/user? thats kind of what I was wondering.

Reply:
@MeredithAnya I mean, it wouldn't hurt. I just think I would look at the table changes instead of log messages.